mysql/assets/configuration/spec.yaml

```diff
@@ -665,6 +665,7 @@ files:
                 description: |
                   Set to `false` to disable the collection of comments in your SQL statements.
                   Requires `collect_metadata: true`.
+                  Note: This option must be `true` in order to propagate APM traces.
```

Comment:
I think `propagate APM traces` is slightly confusing. How about
```
Note: This option must be `true` in order to correlate Database Monitoring samples and APM traces. See https://docs.datadoghq.com/database_monitoring/connect_dbm_and_apm
```

Reply:
I went with propagate because that term was used throughout the setup documentation page linked, but I do think your suggestion sounds clearer. Happy to update